gui/src/controls/textBlock.ts

```diff
@@ -20,6 +20,10 @@ module BABYLON.GUI {
             return this._resizeToFit;
         }
 
+        public set lineSpacing(value: number) {

```

Comment:
Not sure if this is the convention in the GUI, but why is there only a setter and no getter?

Reply:
Added the getter :).